core/templates/dev/head/pages/exploration-editor-page/statistics-tab/services/state-improvement-suggestion.service.spec.ts

```diff
@@ -373,8 +304,8 @@ describe('StateImprovementSuggestionService', function() {
         },
       }
     };
-
-    var _createState = function(destStateName) {
+    // TODO(#7165): Replace any with exact type.
```

Comment:
Why not use the actual type in this PR, here and elsewhere?

I think we talked about this: do not use "any", that should be the (very) rare case. If "any" is ever used, include full justification for why it's impossible to use the correct type now. (The types actually seem pretty clear to me, reading the code below.)

Reply:
Sorry, my bad. I have fixed all the types. Will make sure to avoid this mistake in future.